.gitignore

```diff
@@ -55,7 +55,6 @@ bld/
 
 # Visual Studio 2017 auto generated files
 Generated\ Files/
-**/*Properties/launchSettings.json
```

Comment:
Was this removal on purpose?

Reply:
It was removed because I used `git revert` to auto generate a revert commit for  [eng/pipelines/runtime.yml](https://github.com/dotnet/performance/pull/2314), and it had the change adding it to .gitignore. I think we want to keep it though; I will fix that.